xbmc/cores/VideoPlayer/VideoPlayer.h

```diff
@@ -583,6 +587,9 @@ class CVideoPlayer : public IPlayer, public CThread, public IVideoPlayer,
   bool m_HasVideo;
   bool m_HasAudio;
 
+  std::set<std::string> m_UpdateStreamDetails;
+  CCriticalSection m_Section;
```

Comment:
no locks, VP communicates via messages.

Reply:
Then sould I set m_UpdateStreamDetails with message to VP?